fe/fe-core/src/main/java/com/starrocks/catalog/Type.java

```diff
@@ -582,9 +582,10 @@ public boolean isStringType() {
     // 2. don't support filter
     // 3. don't support group by
     // 4. don't support index
+    // 5. don't support array
     public boolean isOnlyMetricType() {
```

Comment:
shouldn't change here. array could be key column， we also could group by array

Reply:
Thanks, updated